cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -785,6 +785,31 @@ public static String getZKQuorum(CConfiguration cConf) {
     /** Whether to enable entity tagging for metrics for aggregation purposes. */
     public static final String AUTHORIZATION_METRICS_TAGS_ENABLED = "security.authorization.metrics.tags.enabled";
 
+    /**
+     * Default use subscriber config for writer
+     */
+    public static final String USE_SUBSCRIBER_METADATA_KEY = "metrics.use.subscriber.metadata.key";
+
+    /**
+     * Default use sync write with poll freq config for writer
+     */
+    public static final String SYNC_WRITE_WITH_POLL_FREQ = "metrics.sync.write.with.poll.freq";
+
+    /**
+     * Writer specific use subscriber config
+     */
+    public static final String USE_SUBSCRIBER_METADATA_KEY_WRITER = "metrics.use.subscriber.metadata.key.%s";
```

Comment:
should we keep the writer id as a prefix, call cConf.getPropsWithPrefix()?

Reply:
Modified the format to match metrics.writer.writername convention we use for the writer. Not using the cConf.getPropsWithPrefix() since the return value is a Map and does not give the more useful functions like cConf.getBoolean etc.